sdk/python/feast/infra/provider.py

```diff
@@ -20,6 +20,10 @@
 
 
 class Provider(abc.ABC):
+    @abc.abstractmethod
+    def __init__(self, config: RepoConfig):
```

Comment:
Won't we break existing third party providers?

Reply:
Yes.

Reply:
Should we also update the call site to not pass in repo_path when loading custom providers?

Reply:
Yea, good catch.